optuna/integration/lightgbm_tuner/__init__.py

```diff
@@ -25,24 +26,21 @@
 def train(*args: Any, **kwargs: Any) -> Any:
     """Wrapper of LightGBM Training API to tune hyperparameters.
 
-    It tunes important hyperparameters (e.g., `min_child_samples` and `feature_fraction`) in a
-    stepwise manner. Arguments and keyword arguments for `lightgbm.train()
+    It tunes important hyperparameters (e.g., ``min_child_samples`` and ``feature_fraction``) in a
+    stepwise manner. You use it by changing one import statement in your code. Just replace
+    ``import lightgbm as lgb`` with ``import optuna.integration.lightgbm as lgb``. See
```

Comment:
How about the following? It's more compact.

> It is a drop-in replacement for lightgbm.train() // make this a link to the original docs

instead of 

> You can use it by... with ``import optuna.integration.lightgbm as lgb``

Reply:
Thank you. Reflected: https://github.com/optuna/optuna/pull/1217/commits/09e47708d025924b74812e74c28dd7fbc2b30a83.